app/controllers/games_controller.rb

```diff
@@ -1,2 +1,8 @@
 class GamesController < ApplicationController
+	def show
+    @game = Game.find(params[:id])
+    @pieces = @game.pieces
+    @black_player = User.find_by(id: @game.black_uid)
```

Comment:
you already have an association on game for white_user and black_user

you can just write @black_player = @game.black_user

same for white user.